mir_eval/separation.py

```diff
@@ -114,6 +108,12 @@ def validate(reference_sources, estimated_sources):
                                                 MAX_SOURCES))
 
 
+def _any_source_silent(sources):
+    """Returns true if the parameter sources has any silent first dimensions"""
+    return np.any(np.all(np.sum(
+        sources, axis=np.arange(2, sources.ndim)) == 0, axis=1))
```

Comment:
Oops, looks like `np.arange` doesn't work here when `sources.ndim == 2`, my mistake.  Kind of weird and disappointing.  Looks like you will need to go back to tuple(range).